app/packs/src/apps/admin/AdminGroupElement.js

```diff
@@ -175,12 +174,12 @@ export default class AdminGroupElement extends React.Component {
                     <tbody>
                       {groupElement.devices.map((u, i) => (
                         <tr key={`row_${groupElement.id}_${u.id}`} id={`row_${groupElement.id}_${u.id}`} style={{ backgroundColor: '#c4e3f3' }}>
-                          <td className="w-5  py-3 fs-4">{i + 1}</td>
-                          <td className="w-20 py-3 fs-4">{u.name}</td>
-                          <td className="w-10 py-3 fs-4">{u.initials}</td>
-                          <td className="w-20 py-3 fs-4">{ }</td>
-                          <td className="w-15 py-3 fs-4">{ }</td>
-                          <td className="w-30 py-3 fs-4">
+                          <td className="w-5  py-3">{i + 1}</td>
+                          <td className="w-20 py-3">{u.name}</td>
+                          <td className="w-10 py-3">{u.initials}</td>
+                          <td className="w-20 py-3">{ }</td>
+                          <td className="w-15 py-3">{ }</td>
+                          <td className="w-30 py-3">
                             <DeleteGroupDeviceButton rootType={'Group'}
```

Comment:
You can remove the w-5, w-20, w-10, w-15 styles.
They do not exist in bootstrap.
Everything should look the same

Reply:
Yes it was indeed redundant. Made necessary changes.